internal/event/redis.go

```diff
@@ -111,7 +111,7 @@ func (s *SubscriptionRedisImpl) loop() {
 				s.logger.Error(err, "Monitor event is invalid.")
 				continue
 			}
-			s.c <- client.NewMonitorEventOK(&payload)
+			s.c <- client.NewMonitorIngestEventOK(&payload)
```

Comment:
Should we remove "Monitor" from `client.NewMonitorIngestEventOK` too?

Reply:
This is generated code, based on the endpoint name.